client/src/components/Feed/FiltersList.js

```diff
@@ -17,34 +17,34 @@ import { ROYAL_BLUE, WHITE } from "constants/colors";
 
 const TagButton = styled(Button)`
   align-items: center;
-  background-color: ${WHITE};
-  border: 0.1rem solid ${ROYAL_BLUE};
+  background-color: ${theme.button.secondary.backgroundColor};
```

Comment:
Let's remove `background-color, `color`, and `border` to use `${theme.button.tertiary}`. Or maybe we could add all the styling here for the tags in `theme.js`, and then access it like ${theme.button.tag} ?

Reply:
Done.